src/selected_words_counter/extract_files.py

```diff
@@ -0,0 +1,180 @@
+import os
+import re
+import zipfile
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from glob import glob
+
+import extract_msg
+from tqdm import tqdm
+
+from selected_words_counter.functions import process_file
+
+"""
+
+This file is used to store code for extracting text from different file formats.
+
+@author: Michael de Winter
+"""
+
+def process_and_save_file(
+    afilepath,
```

Comment:
This how i like to make my variable names, since filepath would be a easy to overwrite variable name.

For now i think i will be too much work to change this all